lib/di/room/room_di.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:fluffychat/di/base_di.dart';
+import 'package:fluffychat/domain/usecase/create_room_interactor.dart';
+import 'package:fluffychat/domain/usecase/set_room_avatar_interactor.dart';
+import 'package:get_it/get_it.dart';
+import 'package:matrix/matrix.dart';
+
+class RoomDI extends BaseDI {
+  @override
+  String get scopeName => 'Room';
+
+  @override
+  void setUp(GetIt get) {
+    Logs().d('RoomDI::setUp()');
+
+    get.registerFactory<SetRoomAvatarInteractor>(
+      () => SetRoomAvatarInteractor(),

```

Comment:
IMO, `create room` not need it in directly

Reply:
Can you be more specific